app/User.php

```diff
@@ -22,6 +22,13 @@ public function getBackwardsNameAttribute() {
         return $this->lname.', '.$this->fname;
     }
 
+    public function getBackwardsPublicNameAttribute() {
+        if ($this->name_privacy == 1) {
+            return $this->id.', '.$this->fname;
+        }
+        return $this->lname.', '.$this->fname;
+    }
+
```

Comment:
If this is how the ATM wants it, that should be fine. However, I think it would be cleaner to just show the CID and leave out the first name as well. Just my thoughts...

Reply:
@iccowan We're implementing this the same way that the division is. They are also showing CID and first name.

Reply:
If it's been cleared by the division & ATM, then I think this is fine for now. I'll close this out and approve this